scripts/entrypoint.sh

```diff
@@ -14,11 +14,14 @@ if [ -z "${SKIP_API}" ]; then
   mkdir -p api/v2
   mkdir -p api/v3
   echo "Generating API files:"
-  ruby ./scripts/APIv*.rb
```

Comment:
Had to modify this, as the wildcard wasn't working and was only running the v1 script